kobo/settings/base.py

```diff
@@ -844,6 +845,11 @@ def __init__(self, *args, **kwargs):
     'socket_timeout': env.int('REDIS_SESSION_SOCKET_TIMEOUT', 1),
 }
 
+CACHES = {
+    # Set CACHE_URL to override
+    'default': env.cache(default=env.str("REDIS_SESSION_URL", default='redis://redis_cache:6380/2')),
```

Comment:
Seems reasonable. That means adding to kobo-install right? I suppose kobo-install doesn't need to ask anything extra, but needs to set this new env var.

For future local dev environments, it could use the passwordless default redis_cache with a different index.

Reply:
Exactly kobotoolbox/kobo-install#193